package.json

```diff
@@ -21,13 +21,15 @@
   "devDependencies": {
     "babel": "^4.6.6",
     "broccoli-asset-rev": "^2.0.0",
+    "broccoli-sass": "git://github.com/ef4/broccoli-sass",
```

Comment:
I don't think this is necessary as ember-cli-bootstrap-sassy already pulls it in